.github/workflows/publish_timestamped_release_patch.yml

```diff
@@ -0,0 +1,59 @@
+name: Publish Timestamped Snapshot Artifacts
```

Comment:
@udda1996 Shall we use existing workflow for both branches? Maybe we can update the current workflow name without mentioning branch name